### PR TITLE
Wrap account_tx binary parsing in async.setImmediate

### PR DESCRIPTION
**Pull request by wltsmrz**

### CODE REVIEW

File: src/js/ripple/remote.js [hunk 1]
```diff
@@ -1424,10 +1425,19 @@ Remote.prototype.requestAccountTx = function(options, callback) {
...
-    if (options.parseBinary) {
-      res.transactions = res.transactions.map(Remote.parseBinaryTransaction);
+    if (!options.parseBinary) {
+      request.emit('transactions', res);
+      return;
     }
-    request.emit('transactions', res);
+
+    async.mapSeries(res.transactions, function(transaction, next) {
+      async.setImmediate(function() {
```
Comment (geertweening) [inline src/js/ripple/remote.js]: Would be better to replace with `async.nextTick`. Internally `async.setImmediate` is assigned to `async.nextTick`, but the API docs only show `nextTick` as an available method (https://github.com/caolan/async#nextTick)

Comment (wltsmrz) [inline src/js/ripple/remote.js]: Ah. I didn't read the docs, I read [the code](https://github.com/caolan/async/blob/master/lib/async.js#L86-L113). Internally async seems to [prefer](https://github.com/caolan/async/blob/master/lib/async.js#L553) setImmediate. I also prefer to use setImmediate. I'll see if I can get it documented on the async readme so we can use it instead. Leave this pull request open until then

Comment (geertweening) [inline src/js/ripple/remote.js]: cool

Comment (wltsmrz) [inline src/js/ripple/remote.js]: https://github.com/caolan/async/pull/692

Comment (beaugunderson) [inline src/js/ripple/remote.js]: (async maintainer here); just merged that PR. :)

Comment (wltsmrz) [inline src/js/ripple/remote.js]: Thanks!

**Pull request merged**